rust/connlib/snownet/tests/lib.rs

```diff
@@ -210,9 +210,10 @@ fn migrate_connection_to_new_relay() {
         ),
     )];
     alice = alice.with_relays("alice", HashSet::from([1]), &mut relays, clock.now);
+    bob = bob.with_relays("bob", HashSet::from([1]), &mut relays, clock.now);
```

Comment:
Same here, this could be isolated. The portal will notify both the client and the gateway of a disconnected relay so we should mimic this in the test.

Reply:
👍